.github/workflows/tests.yml

```
@@ -41,6 +41,13 @@ jobs:
       run: |
         pipenv sync --dev
 
+    - name: Install frontend dependencies
+      if: "exists('app/package.json')"
```

actionlint (1.7.7)

undefined function "exists". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)